ff_dir.c

```diff
@@ -2886,7 +2886,7 @@ FF_Error_t FF_ExtendDirectory( FF_IOManager_t * pxIOManager,
 /* Windows says: don't use these characters: '\/:*?"<>|'
  *  "     *     /	    :     <     >     ?    '\'    ?     | */
     0x22, 0x2A, 0x2F, 0x3A, 0x3C, 0x3E, 0x3F, 0x5C, 0x7F, 0x7C
-};
+}
```

Comment:
Uncrustify removed the semicolon. 
I agree, it looks nicer without it, but now it doesn't build.